climate-disasters.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+import numpy as np
+
+df = pd.read_csv('~/dev/climate-datasets/files/emdat_africa_1992-2023_natural.csv', encoding = 'unicode_escape', engine ='python')
```

Comment:
The `~/dev/climate-datasets/files` is specific to your machine, and Linux style file systems, rather use local `./files`